internal/store/store.go

```diff
@@ -484,35 +488,119 @@ func (s *Store) PushCompose(composeID uuid.UUID, bp *blueprint.Blueprint, checks
 		repos = append(repos, source.RepoConfig())
 	}
 
-	size = s.distro.GetSizeForOutputType(composeType, size)
```

Comment:
It seems you accidentally removed this line - probably while rebasing. The removal of this line breaks the rhel 8.2 for me - without it the default image size isn't set, which leads to images with size of 0.

Reply:
With this fix I was able to succesfully build an image! :slightly_smiling_face:

Reply:
thanks!